### PR TITLE
BL-4909-privacy-policy

### PR DESCRIPTION
**Pull request by Steveybrown**

### CODE REVIEW

File: webapp/src/main/resources/template/privacy-policy.hbs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        </h1>
+    </header>
+    <div class="text">
+        <p>We collect certain information or data about you when you use MOT
+            reminders.</p>
+        <p>This includes:</p>
+        <ul class="list-bullet">
+            <li>questions, queries or feedback you leave</li>
+            <li>your IP address, and details of which version of web browser you used</li>
+            <li>information on how you use the site, using cookies and page
```
Comment (paddyfox) [inline webapp/src/main/resources/template/privacy-policy.hbs]: Web browser should externally link to: https://www.gov.uk/help/browsers
Cookies should internally link to Cookies Page

File: webapp/src/main/resources/template/privacy-policy.hbs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            tagging techniques to help us improve the website</li>
+            <li>your email address to allow us to send you a MOT reminder and
```
Comment (paddyfox) [inline webapp/src/main/resources/template/privacy-policy.hbs]: a MOT reminder

should be: an MOT reminder

File: webapp/src/main/resources/template/privacy-policy.hbs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+        <h2 class="heading-medium">Keeping your data secure</h2>
+        <p>Transmitting information over the internet is generally not completely
+            secure, and we can’t guarantee the security of your data</p>
+        <p>Any data you transmit is at your own risk.</p>
+        <p>We have procedures and security features in place to try and keep your
+            data secure once we receive it.</p>
+        <p>We won’t share your information with any other organisations for
+            marketing, market research or commercial purposes, and we don’t pass
+            on your details to other websites</p>
```
Comment (paddyfox) [inline webapp/src/main/resources/template/privacy-policy.hbs]: Full stop missing here

File: webapp/src/main/java/uk/gov/dvsa/motr/web/resource/PrivacyPolicyResource.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+
+import static java.util.Collections.emptyMap;
+
+@Singleton
+@Path("/privacy-policy")
+@Produces("text/html")
+public class PrivacyPolicyResource {
```
Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/resource/PrivacyPolicyResource.java]: Unit tests for this class for completeness :)

**Comment by aaronlong** [comment]
Do we need selenium coverage for this story?

File: selenium/src/test/java/uk/gov/dvsa/motr/journey/MotReminderTests.java [hunk 5]
```diff
@@ -111,6 +112,17 @@ public void canViewTermsAndConditionsPageWhenClickingLinkInFooter() {
...
     }
 
+    @Test(description = "As a user of the site with a vested interest in the privacy policy of the service, I can view them")
+    public void canViewPrivacyPageWhenClickingLinkInFooter() {
+
+        //Given I am a user of the site
+        //When I click the privacy policy link in footer of the page
+        PrivacyPage privacyPage = motReminder.clickPrivacyPolicyLink();
+
+        //Then I am taken to the terms and conditions page
```
Comment (paddyfox) [inline selenium/src/test/java/uk/gov/dvsa/motr/journey/MotReminderTests.java]: Comment should ready Privacy Policy, not terms and conditions

File: webapp/src/test/java/uk/gov/dvsa/motr/web/resource/PrivacyPolicyTest.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+import org.junit.Test;
+
+import uk.gov.dvsa.motr.web.test.render.TemplateEngineStub;
+
+import static org.junit.Assert.assertEquals;
+
+import static uk.gov.dvsa.motr.web.test.render.TemplateEngineStub.RESPONSE;
+
+public class PrivacyPolicyTest {
+    @Test
```
Comment (paddyfox) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/resource/PrivacyPolicyTest.java]: New line

**Pull request merged**